bigquery/main.go

```diff
@@ -205,8 +205,13 @@ func (n *bqNotifier) SendNotification(ctx context.Context, build *cbpb.Build) er
 		return nil
 	}
 	buildImages := []*buildImage{}
+	shaSet := make(map[string]bool)
```

Comment:
Please write a test for this change if possible.

Reply:
prototyped a bit to see if it was possible, but would require too much core bq notifier code change, so leaving as it is